src/aks-preview/azext_aks_preview/_helpers.py

```diff
@@ -88,4 +88,6 @@ def _trim_fqdn_name_containing_hcp(normalized_fqdn: str) -> str:
     attached
     """
     storage_name_without_hcp, _, _ = normalized_fqdn.partition('-hcp-')
+    if len(storage_name_without_hcp) > 63:
```

Comment:
how's the 63 coming from, just curious

Reply:
from the docs here: https://docs.microsoft.com/en-us/rest/api/storageservices/naming-and-referencing-containers--blobs--and-metadata#container-names.

Reply:
Can we define a constant for it and add the link to the code comment ?

Reply:
added.